package.json

```diff
@@ -121,7 +122,8 @@
     "postpack": "rm -f oclif.manifest.json",
     "prepack": "npm run clean && npm run compile && oclif-dev manifest && oclif-dev readme",
     "pretest": "npm run lint && npm run compile && npm run depcheck",
-    "lint": "eslint \"**/*.ts\" --quiet --fix",
+    "lint": "npm run lint:dev -- --quiet",
```

Comment:
Will this now run against compiled javascript code? Why drop the ts filter here?

Reply:
As far as I can tell, `eslint "**/*.ts"` and `eslint . --ext .ts` are equivalent. Went with the latter for consistency with commerce-sdk and because I think avoiding the escaped quotes makes it easier to read in the package.json.

Reply:
Yes...but you didn't add --ext ts to this command.  Only the dev version.

Reply:
nevermind I see what you are doing now.